app/src/main/res/layout/progress_bar_layout.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <data>
+        <import type="android.view.View"/>
+        <variable
+            name="isVisible"
+            type="boolean"/>
+    </data>
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:clickable="true"
+        android:focusable="true"
+        android:elevation="@dimen/max_elevation"
+        android:visibility="@{isVisible ? View.VISIBLE : View.GONE}">
```

Comment:
Cleaner to use `isVisible` from `XmlBindingAdapters`, we do this in other layouts already.

Reply:
Would I be able to use that since this is a standalone reusable XML file and doesn't have any associated fragment/ViewModels?